pkg/pebinaryexecutor/pebinary.go

```diff
@@ -190,6 +200,19 @@ func (p *PEBinaryConfig) KubeProxy(args []string) error {
 	}
 
 	args = append(getArgs(extraArgs), args...)
+	var si int
+	for i, arg := range args {
+		if strings.Contains(arg, "source-vip") {
+			si = i
+		}
+	}
+
+	si, err := argIndex(args, "source-vip")
+	if err != nil {
+		return err
+	}
+	args[si] = fmt.Sprintf("--source-vip=%s", vip)
```

Comment:
Nit:

```suggestion
	args[si] = "--source-vip="+vip
```

This is more direct and removes a significant amount of unnecessary processing from "under the hood".